indexer/indexer.go

```diff
@@ -182,6 +184,10 @@ func defaultBadgerOptions(
 	// filters will be immediately discarded from the cache).
 	opts.LoadBloomsOnOpen = false
 
+	opts.BlockSize = 1 << blockLeftShift // 1MB
+
+	opts.NumCompactors = 2
```

Comment:
what does these new flag do? would be great to have a comment here explaining that.